packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -333,6 +355,14 @@ class _AnimatedContainerState extends AnimatedWidgetBaseState<AnimatedContainer>
 ///
 /// Only works if it's the child of a [Stack].
 class AnimatedPositioned extends ImplicitlyAnimatedWidget {
+  /// Creates a widget that animates its position implicitly.
+  ///
+  /// Only two out of the three horizontal values ([left], [right],
+  /// [width]), and only two out of the three vertical values ([top],
+  /// [bottom], [height]), can be set. In each case, at least one of
+  /// the three must be null.
```

Comment:
I'm not sure.  Maybe "overdescribed" would be better to avoid implicating constraints (as in box constraints).